ts/src/bitfinex2.ts

```diff
@@ -3174,7 +3174,7 @@ export default class bitfinex2 extends Exchange {
         return reversedArray as FundingRateHistory[];
     }
 
-    parseFundingRate (contract, market: Market = undefined) {
+    parseFundingRate (contract: Dict, market: Market = undefined): FundingRate {
```

Comment:
@samgermain as we discussed before, this is not correct since bitfinex2 provides an array